### PR TITLE
Add ungrouped destination chart

### PR DESCRIPTION
**Pull request by IvanM26**
A new chart was added to Living Situation page. This chart shows the number of youth by destination. The difference with the pie chart ("at Exit") is that this new chart shows the categories without grouping. To make the distinction clear, titles were updated accordingly. As part of this task the title in Education page was updated as well to keep consistency throughout the app codebase. As a final note, some packages were updated.

### CODE REVIEW

File: R/mod_living_situation.R [hunk 1]
```diff
@@ -89,6 +89,24 @@ mod_living_situation_ui <- function(id){
...
     ),
 
+    shiny::fluidRow(
+      shiny::column(
+        width = 12,
+
+        bs4Dash::box(
+          title = "# of Youth by Destination (at Exit)",
+          width = NULL,
+          height = "680px",
```
Comment (mthomas-ketchbrook) [inline R/mod_living_situation.R:99]: I assume this box's height is taller than the `DEFAULT_BOX_HEIGHT` to accommodate the large number of categories on the y-axis?

Comment (mthomas-ketchbrook) [reply R/mod_living_situation.R:99]: The y-axis labels are very long, which makes the the x-axis labels too close together and takes away from the data in the chart (bars) itself.  Let's consider how we can improve this (perhaps by trimming some of the labels).
image

File: R/mod_living_situation.R [hunk 2]
```diff
@@ -318,7 +333,36 @@ mod_living_situation_server <- function(id, project_data, enrollment_data, exit_
...
+        )
+
+      out |>
+        dplyr::count(destination) |>
+        dplyr::arrange(n) |>
+        dplyr::filter(
+          !is.na(destination),
+          !destination %in% c("Client doesn't know",
+                              "Data not collected",
+                              "Client prefers not to answer")
```
Comment (mthomas-ketchbrook) [inline R/mod_living_situation.R:345]: We should also remove "No exit interview completed" (see chart above) -- I believe that would belong in the "Data Quality Statistics" section. @trevinflick do you agree?

File: R/mod_education.R
Comment (mthomas-ketchbrook) [inline R/mod_education.R:1]: We should add "Group" to the box title for the sankey chart, as well.

**Review by mthomas-ketchbrook — CHANGES_REQUESTED** [review]
Hi @IvanM26 -- great work! I'm recommending a few small changes, and we should brainstorm on how to better display the new horizontal bar chart for "Living Situation" so that the bars/labels aren't so condensed.

File: R/mod_living_situation.R (hunk 1 above)
Comment (IvanM26) [reply R/mod_living_situation.R:99]: Regarding the first comment, it is like you say! I needed to increase the chart height due to the number of different categories.

Regarding the second comment, I have trimmed the label text. Long labels now end with an ellipsis. Users can see the full name by hovering over the bar.

File: R/mod_living_situation.R (hunk 2 above)
Comment (IvanM26) [reply R/mod_living_situation.R:345]: As you mention, that value should not be shown in the chart. I see that the other charts are removing that category (as well as `Worker unable to determine`). So I will add both categories to the filter.

File: R/mod_education.R
Comment (IvanM26) [reply R/mod_education.R:1]: Done!

**Review by mthomas-ketchbrook — APPROVED** [review]

**Pull request merged**